openpype/hosts/maya/api/lib.py

```diff
@@ -1145,9 +1213,15 @@ def extract_alembic(file,
     # Alembic Exporter requires forward slashes
     file = file.replace('\\', '/')
 
+    # Ensure list arguments are valid.
+    args = [attr, attrPrefix, root]
+    for arg in args:
+        if arg is None:
+            arg = []
```

Comment:
This does not do what you want - it does nothing since it doesn't change the actual arguments at all. It's unfortunate that the arguments are now keyword arguments and we can't really access them by the string key (unless we use `locals()` maybe?)

```python
for key, key_type in _alembic_options.items():
    if key_type == (list, tuple):
        if locals()[key] is None:
            locals()[key] = []
```